packages/react-scripts/scripts/init.js

```diff
@@ -36,12 +36,7 @@ module.exports = function(
   appPackage.dependencies = appPackage.dependencies || {};
 
   // Setup the script rules
-  appPackage.scripts = {
-    start: 'react-scripts start',
-    build: 'react-scripts build',
-    test: 'react-scripts test --env=jsdom',
-    eject: 'react-scripts eject',
-  };
+  appPackage.scripts = require('./utils/scripts');
```

Comment:
Generation of the `scripts` section inside the `package.json` for the template now based on the `./utils/scripts` content.